### DISCUSSION TITLE
Integrate postgraphile as library

### DISCUSSION THREAD

**Discussion by iarkaroy** [adonisjs/core#4136] [Help]
I am looking to integrate https://www.graphile.org/postgraphile into my adonis app. Can anyone share any idea on how that can be done?

Thanks

**Comment by Julien-R44** [comment]
I'm not familiar with Postgraphile but I guess you'll have to write your own integration plugin. 

https://www.graphile.org/postgraphile/usage-library/

Feel free to share the problems you might encounter during development
